exercises/concept/intro-select/intro-select_test.sql

```diff
@@ -0,0 +1,16 @@
+-- Create database:
+.read ./create_fixture.sql
+
+.mode columns
+
+-- Generate expected output
+.output expected_output.txt
+.read ./intro-select_exemplar.sql
+
+-- Run user solution
+.output user_output.txt
+.read ./intro-select.sql
+
+-- Compare expected vs actual
+.shell diff expected_output.txt user_output.txt
```

Comment:
The diff output is certainly not user-friendly.  Instead of `diff`, we could run a custom Python script here that would generate a friendlier diff.

Reply:
Either way, note that this would work with the existing test runner.